app/src/main/java/org/oppia/android/app/player/audio/AudioFragmentPresenter.kt

```diff
@@ -266,6 +272,7 @@ class AudioFragmentPresenter @Inject constructor(
   }
 
   private fun hideAudioFragment() {
+    isPauseAudioRequestPending = true
```

Comment:
I'm not sure that I understand the flow fully. The ``(fragment as AudioUiManager).pauseAudio()`` line below seems to just call through to ``pauseAudio`` above, so it's a bit unclear to me why we only set this to true here but not in other cases when a pause seems to be requested (such as when 'pauseAudio' itself is clicked). Am I maybe missing something?

Reply:
@BenHenning updating the status of "isPauseAudioRequestPending" inside pauseAudio works the same. What i wanted was to update this early enough when the process of hiding the audioPlayer Fragment has started this is on "hideAudioFragment". This seems to fix the issue according to what i found out.